ipi/engine/normalmodes.py

```diff
@@ -87,6 +87,8 @@ def __init__(
         bosons=None,
         dt=1.0,
         nmts=1,
+        fft_threads=1,
+        fft_float32=False,
```

Comment:
isn't it more efficient to do float32? And if previously things were hardcoded to single precision (float32), as mentioned on slack, then I guess it was tested to provide reasonable accuracy?

Reply:
no it was not tested I think. this goes back to the very early days. there is a small overhead from doing float64 but it reduces noticeably the drift of the conserved quantity so I think it should be the default